template-parts/updatebox-section.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Climate Hub partners page supplementary updatebox section template
+ * @package climatehub
+ */
+
+
+  if(get_field('enable_custom_field')): ?>
+    <div class="container">
+      <section>
+        <div class="row">
+          <div class="col-sm">
+                  <?php if(get_field('title')): ?>
+                   <h2><?php echo get_field('title') ?></h2>
+                  <?php endif; ?>
+                  
+                  <?php if( get_field('project_description') ): ?>
```

Comment:
I would do the same thing here, because `project_description` might be used on the projects page.